### DISCUSSION TITLE
How to use splade with the pre-tokenized file?

### DISCUSSION THREAD

**Discussion by Wenze7** [castorini/pyserini#1361] [Q&A] [answered]
Hi, i'm glad to see this tools!
I want to reproduce splade, but i'm using paddlepaddle instead of pytorch, so i want to ask for how can i use the jsonl file to search instead of using a pytorch-model to be the encoder?

**Comment by Wenze7** [comment]
I have noticed the following scirpt:
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/55132749/204213408-1b223bf1-311d-4430-9a77-cb245b9dc1e1.png">
I want to know the format of msmarco-passage-dev-subset-distill-splade-max, could you give me the url of this file?

**Comment by Wenze7** [comment, marked as answer]
@lintool Hi! could you give me some help? I'm very worried, thanks so much!!!

**Reply by MXueguang** [reply]
Hi, the encoded vectors is here
https://github.com/castorini/anserini/blob/master/src/main/resources/topics-and-qrels/topics.msmarco-passage.dev-subset.distill-splade-max.tsv.gz

**Reply by Wenze7** [reply]
Thanks very much!
